### PR TITLE
fix #74895 rojadirecta.fr

### LINKED ISSUE #74895: rojadirecta.fr
### Issue URL (Ads)
 [https://www.rojadirecta.fr/ver/rumania-liga-1-poli-iasi-vs-astra/canal-55.php](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.rojadirecta.fr%2Fver%2Frumania-liga-1-poli-iasi-vs-astra%2Fcanal-55.php)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 17
Browser: | Opera
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Spanish/Portuguese
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#74895

### CODE REVIEW

File: FrenchFilter/sections/adservers.txt [hunk 1]
```diff
@@ -1,8 +1,9 @@
 !
 ! Section contains list of advertising networks
 !
+||dustymural.com^$third-party
```
Comment (Alex-302) [inline FrenchFilter/sections/adservers.txt:4]: duplicate

File: FrenchFilter/sections/adservers.txt [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 ! Section contains list of advertising networks
 !
+||dustymural.com^$third-party
 ||mystighty.info^
 ||i4tvy8vk9f74.com^$third-party
 ||sohjah-thahka.info^
 ||goitcl.com^
-||mineralisasse.site^$third-party
\ No newline at end of file
+||mineralisasse.site^$third-party
```
Comment (Alex-302) [inline FrenchFilter/sections/adservers.txt:9]: I don't see it

File: FrenchFilter/sections/specific.txt [hunk 3]
```diff
@@ -2,6 +2,7 @@
 !-------------------- Specific rules --------------------!
 !--------------------------------------------------------!
 !
+rojadirecta.fr,veopartidos.net##.overlay
```
Comment (Alex-302) [inline FrenchFilter/sections/specific.txt:5]: not reproduced

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: FrenchFilter/sections/adservers.txt (hunk 2 above)
Comment (wout-codes) [reply FrenchFilter/sections/adservers.txt:9]: I didn't add that. It went down after adding to ad server.

File: FrenchFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply FrenchFilter/sections/specific.txt:5]: Ok let met send screenshot.

File: FrenchFilter/sections/adservers.txt (hunk 1 above)
Comment (wout-codes) [reply FrenchFilter/sections/adservers.txt:4]: Ok deleted.

File: FrenchFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply FrenchFilter/sections/specific.txt:5]: image
Link: `https://www.rojadirecta.fr/ver/espana-laliga-santander-levante-vs-atletico-de-madrid/espn-2-en-vivo-por-internet.php`

Comment (wout-codes) [reply FrenchFilter/sections/specific.txt:5]: @Alex-302

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**